numpy/tests/test_matlib.py

```diff
@@ -28,10 +28,19 @@ def test_identity():
     assert_array_equal(x, np.matrix([[1, 0], [0, 1]]))
 
 def test_eye():
-    x = numpy.matlib.eye(3, k=1, dtype=int)
-    assert_array_equal(x, np.matrix([[ 0,  1,  0],
-                                     [ 0,  0,  1],
-                                     [ 0,  0,  0]]))
+    xc = numpy.matlib.eye(3, k=1, dtype=int)
+    assert_array_equal(xc, np.matrix([[ 0,  1,  0],
+                                      [ 0,  0,  1],
+                                      [ 0,  0,  0]]))
+    assert xc.flags.c_contiguous
+    assert not xc.flags.f_contiguous
+
+    xf = numpy.matlib.eye(3, 4, dtype=int, order='F')
+    assert_array_equal(x, np.matrix([[ 1,  0,  0,  0],
```

Comment:
NameError: global name 'x' is not defined

Reply:
MERP! Copy-pasta (was in a hurry last night and didn't run tests locally, mea culpa).